src/test/routes/idam-signin.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+
+describe('idam signin page', () => {
+  describe('on GET', () => {
+    test('should return otp-login page', async () => {
+      await request(app)
+        .get('/idam-signin')
+        .expect((res) => expect(res.status).to.equal(200));
+    });
+  });
+
+  describe('on POST', () => {
+    test('should return subscription management page', async () => {
```

Comment:
The above description needs updating to reflect the returned page (same on the next test)